scripts/nbconverted/explore_HighWeight_genes.py

```diff
@@ -0,0 +1,242 @@
+
+# coding: utf-8
+
+# # Explore latent space features
+# We want to know if our VAE model is capturing our signal
+# 
+# Identify high weight (HW) genes for each latent space feature (node)
+# Determine if genes in group A and B are found within these highly weighted genes
+
+# In[2]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+import os
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from scipy.stats import pearsonr, spearmanr
+
+
+randomState = 123
+from numpy.random import seed
+seed(randomState)
+
+
+# In[3]:
+
+
+# Load arguments
+analysis_name = "sim_AB_2775_300_v2"
+data_file = os.path.join(os.path.dirname(os.getcwd()), "data", analysis_name, "VAE_weight_matrix.txt")
+A_file = os.path.join(os.path.dirname(os.getcwd()), "data", analysis_name, "geneSetA.txt")
+B_file = os.path.join(os.path.dirname(os.getcwd()), "data", analysis_name, "geneSetB.txt")
+
+#HW_file = os.path.join(os.path.dirname(os.getcwd()), "output", analysis_name, "HW_genes.txt")
+
+
+# In[4]:
+
+
+# Read in data
+geneSetA = pd.read_table(A_file, header=0, index_col=0)
+geneSetB = pd.read_table(B_file, header=0, index_col=0)
+
+weight = pd.read_table(data_file, header=0, index_col=0).T
+weight.head(5)
+
+
+# In[5]:
+
+
+# Plot the distribution of gene weights per latent feature
+#num_features = weight.shape[1]
```

Comment:
Here I was trying to visualize the distribution of weights for all 300 latent space features, but I had trouble trying to figure out the plot something so large.  I think its something I would like to still look into in the future, which is why it is commented here instead of just being deleted.  It was taking a bit too long and I felt that my time might be better spent on some of the other analyses temporarily